tests/integration/plots/test_plots.py

```diff
@@ -48,9 +48,11 @@ def extract_vega_specs(html_path, plots_ids):
 
     reader = BeautifulSoup(content, features="html.parser")
     for plot_id in plots_ids:
-        clean_id = Renderer.remove_special_chars(plot_id)
-        div_id = f"plot_{clean_id}"
-        script = _remove_blanks(reader.find("div", id=div_id).script.text)
+        script = _remove_blanks(
```

Comment:
@pared There are some tests here marked as vscode but vscode doesn't rely on the generated HTML (afaik). Maybe we should separate to only mark with vscode the relevant logic?

Reply:
Yep, it will be part of #7956